plans/podman_e2e_test.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -eox pipefail
+
+cat /etc/redhat-release
+rpm -q container-selinux golang podman
+if [ -f /etc/fedora-release ]; then
+    mount -o remount,size=10G /tmp
+fi
+dnf --disablerepo=* --enablerepo=copr:copr.fedorainfracloud.org:rhcontainerbot:podman-next download --source podman
+rpm2cpio podman*.src.rpm | cpio -di
+tar zxf podman-*-dev.tar.gz
+cd podman-*-dev/test/e2e
+PODMAN_BINARY=/usr/bin/podman go test -v config.go config_amd64.go common_test.go libpod_suite_test.go run_selinux_test.go
```

Comment:
@edsantiago @giuseppe if we need podman e2e tests for crun, please let me know the right files to use here. Currently, it's just a copy-paste from what we have for container-selinux.

Reply:
maybe `run_test.go run_userns_test.go run_cgroup_parent.go` ?

Reply:
would this be correct usage:
`PODMAN_BINARY=/usr/bin/podman go test -v config.go config_amd64.go common_test.go libpod_suite_test.go run_test.go run_userns_test.go run_cgroup_parent_test.go`

I see I'll need to test this separately for arm64, but that's a separate issue.